installer/debian/preinst

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+PKG_NAME=@PKG_NAME@
+VERSION="$2"
+
+if test "$1" = install &&
+    dpkg-query --list 'algorand*' > /dev/null
+then
+    if PKG_INFO=$(dpkg-query --show --showformat='${Package} ${Status}\n' 'algorand*' | grep "install ok installed")
```

Comment:
I think this will not catch partial installs / install failures, but that's okay for this iteration.

Reply:
If the install fails or is aborted, I think that will be ok as this is only flagged if a different packages is attempted to be installed.